ci-operator/config/openshift/microshift/openshift-microshift-release-4.13.yaml

```diff
@@ -115,115 +124,43 @@ tests:
   cluster: build02
   run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile|.*\.sh)
   steps:
-    cluster_profile: gcp
+    cluster_profile: aws
     env:
-      GOOGLE_COMPUTE_IMAGE_NAME: rhel-9-v20230203
-    workflow: openshift-microshift-e2e-openshift-conformance-reduced
+      EC2_AMI: ami-07f5ac8ddbcfbb96e
+    workflow: openshift-microshift-e2e-openshift-conformance-reduced-aws
 - as: e2e-openshift-conformance-reduced-arm
   cluster: build02
   optional: true
   run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile|.*\.sh)
   steps:
-    cluster_profile: gcp
-    env:
-      GOOGLE_COMPUTE_IMAGE_NAME: rhel-9-arm64-v20230306
-      GOOGLE_COMPUTE_MACHINE_TYPE: t2a-standard-4
-    workflow: openshift-microshift-e2e-openshift-conformance-reduced
-- as: e2e-reboot
-  cluster: build02
-  run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
-  steps:
-    cluster_profile: gcp
-    env:
-      GOOGLE_COMPUTE_IMAGE_NAME: rhel-9-v20230203
-    workflow: openshift-microshift-e2e-reboot-tests
-- as: e2e-reboot-arm
-  cluster: build02
-  optional: true
-  run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
-  steps:
-    cluster_profile: gcp
-    env:
-      GOOGLE_COMPUTE_IMAGE_NAME: rhel-9-arm64-v20230306
-      GOOGLE_COMPUTE_MACHINE_TYPE: t2a-standard-4
-    workflow: openshift-microshift-e2e-reboot-tests
-- as: e2e-greenboot
-  cluster: build02
-  optional: true
-  run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|scripts/.*|vendor/.*|go\.sum|go\.mod|Makefile)
-  steps:
-    cluster_profile: gcp
-    env:
-      GOOGLE_COMPUTE_IMAGE_NAME: rhel-9-v20230203
-    workflow: openshift-microshift-e2e-greenboot
-- as: e2e-greenboot-arm
-  cluster: build02
-  optional: true
-  run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|scripts/.*|vendor/.*|go\.sum|go\.mod|Makefile)
-  steps:
-    cluster_profile: gcp
+    cluster_profile: aws
     env:
-      GOOGLE_COMPUTE_IMAGE_NAME: rhel-9-arm64-v20230306
-      GOOGLE_COMPUTE_MACHINE_TYPE: t2a-standard-4
-    workflow: openshift-microshift-e2e-greenboot
-- as: e2e-router-smoke-test
+      EC2_AMI: ami-0c231be9fb222349e
+      EC2_INSTANCE_TYPE: a1.2xlarge
+    workflow: openshift-microshift-e2e-openshift-conformance-reduced-aws
+- as: microshift-e2e
```

Comment:
This test only exists from 4.14. You need to keep the reboot, greenboot, load-balancer and router smoke tests in this branch.